### DISCUSSION TITLE
"GitHub Copilot requires VS Code 1.57+. Please update your VS Code"

### DISCUSSION THREAD

**Discussion by didvc** [community/community#65726] [Copilot Conversations] [answered]
### Select Topic Area

Product Feedback

### Body

Tried on VSCode v1.79.2 (system installed) and later v1.81.1 (user installed) the both ways didn't work as the error message pops up like the attached screenshot below. How can I use GitHub Copilot on VSCode on my PC? That's all what I need to know. Thanks.

![image](https://github.com/community/community/assets/24733624/cf1a65af-483b-4aab-a095-2ef7ffbcf14b)

**Comment by tpope** [comment, marked as answer]
Please try updating the GitHub Copilot extension as well.

**Reply by didvc** [reply]
I didn't notice it was even upgradable since it was the first time to install the Copilot extension. Anyways it solved the issue magically, thanks.
